contentcuration/contentcuration/viewsets/feedback.py

```diff
@@ -48,6 +50,54 @@ class Meta:
         fields = BaseFeedbackSerializer.Meta.fields + BaseFeedbackEventSerializer.Meta.fields + BaseFeedbackInteractionEventSerializer.Meta.fields
 
 
+class RecommendationsInteractionEventSerializer(BaseFeedbackSerializer, BaseFeedbackInteractionEventSerializer):
+    recommendation_event_id = serializers.UUIDField()
+
+    class Meta:
+        model = RecommendationsInteractionEvent
+        fields = BaseFeedbackSerializer.Meta.fields + BaseFeedbackInteractionEventSerializer.Meta.fields + ['recommendation_event_id']
+
+    def create(self, validated_data):
+        return RecommendationsInteractionEvent.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        for attr, value in validated_data.items():
+            setattr(instance, attr, value)
+        instance.save()
+        return instance
```

Comment:
I believe the underlying `serializers.ModelSerializer` defines this functionality already. Is there intent that this does something differently, that I'm not seeing?